Client/Frontend/SettingsTableViewController.swift

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
Nit: Indentation (if you're interested, I made a [template](https://www.dropbox.com/s/9l38op33lzhx408/Templates.zip?dl=0) that you can extract to `~/Library/Developer/Xcode` which includes the Moz header in all new Swift files).